README.md

```diff
@@ -1,2 +1,142 @@
-# consume-more-stuff
-A custom CMS for personal product sale
+# Consume More Stuff
+> A custom CMS for personal product sale
+
+## Introduction
+Build a Content Management System using:
```

Comment:
I would probably reiterate in instructions that the data schema needs to be approved by a senior developer to simulate the whole "experience".